frontend/src/metabase/visualizations/visualizations/Text/Text.styled.jsx

```diff
@@ -0,0 +1,52 @@
+import styled from "@emotion/styled";
+import { color } from "metabase/lib/colors";
+
+export const EditModeContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  justify-content: center;
+  overflow: hidden;
+  padding: 0.65em;
+  width: 100%;
+  pointer-events: auto;
+
+  ${({ isPreviewing }) => !isPreviewing && "padding: 0.5em 0.75em;"}
+`;
+
+export const TextInput = styled.textarea`
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  flex: 1 0 auto;
+  background-color: ${color("bg-light")};
+  border: none;
+  border-radius: 8px;
+  box-shadow: none;
+  font-size: 1.143em;
+  height: inherit;
+  line-height: 1.602em;
```

Comment:
Here and in other places you added very precise and unusual values we don't have anywhere else. Usually, we use `rem` with the step of `0.25` (rarely `0.125`)

Reply:
These specific values were just ported over from converting part of this file to styled-components from CSS modules. I did not choose these values in particular as they already existed, but I will check other places I created new styles and update appropriately.

Thank you for the heads up!